doc/code/WatchFaces.md

```diff
@@ -0,0 +1,135 @@
+# Add Watchfaces 
+
+This page will help you to add a new Watch Face to InfiniTime.
+
+## Theory
+
+This example uses the existing Digital Watch Face, copies this to a new watch face as a starting point to design a new additional watch face.
+
+## Files 
+
+- displayapp/UserApps.h
+- displayapp/apps/Apps.h.in
+- displayapp/apps/CMakeLists.txt
+- displayapp/screens/WatchFaceNew.cpp 
+- displayapp/screens/WatchFaceNew.h 
+- components/settings/Settings.h
+- CMakeLists.txt
+
+### Check out and Branch and Copy 
+
+Use git to fork and get a local copy (commands not shown). 
+
+Then with all the submodules updated add a branch and copy the watch face files
+
+```bash
+git submodule update --init
+git checkout -b Watchface_New
```

Comment:
the modern more explicit way is to use git switch

```suggestion
git switch -c Watchface_New
```

Reply:
I updated the git usage here to use git switch as suggested.